frontend/src/components/MainLayout/MainLayout.tsx

```diff
@@ -0,0 +1,22 @@
+import { Outlet } from "react-router";
+import styles from "./MainLayout.module.css";
+
+export default function MainLayout() {
+  return (
+    <main className={styles.main}>
+      <nav className={styles.nav}>
+        <div className={styles.navLeft}>
+          <a className={styles.link}>Home</a>
```

Comment:
We could potentially replace these links with [NavLinks](https://reactrouter.com/en/main/components/nav-link) later.